zuul.d/nodeset.yaml

```diff
@@ -99,6 +99,34 @@
       - name: ocps
         nodes:
           - crc
+      - name: rh-subscription
+        nodes:
+          - standalone
+
+- nodeset:
+    name: centos-9-multinode-rhel-9-2-crc-extracted-2.30-3xl
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo
+      - name: crc
+        label: coreos-crc-extracted-2-30-0-3xl
+      - name: undercloud
+        label: cloud-rhel-9-2
+      # TODO(marios): uncomment nodes as needed
+      # - name: tripleo-controller-1
+      #  label: cloud-rhel-9-2
+      # - name: tripleo-compute-1
+      #  label: cloud-rhel-9-2
+      # - name: tripleo-controller-2
+      #   label: cloud-rhel-9-2
+      # - name: tripleo-controller-3
+      #   label: cloud-rhel-9-2
+    groups:
```

Comment:
Care to add an `osps` group containing the crc node?

Reply:
ack thanks updating as discussed